api/singleLottery.ts

```diff
@@ -0,0 +1,75 @@
+import { NowRequest, NowResponse } from "@now/node";
```

Comment:
```suggestion
import { NowRequest, NowResponse } from "@vercel/node";
```

Reply:
wops looks like a merge problem :( i changed it in the other pr